baselines/dom.generated.d.ts

```diff
@@ -6327,7 +6327,7 @@ declare var DOMRectReadOnly: {
  */
 interface DOMStringList {
     /**
-     * Returns the number of strings in strings.
+     * Returns the number of strings in a list of DOMString (strings).
```

Comment:
This is not really a typo but a rephrase, can you split this in a separate PR at least?

Reply:
Oh, okay, would close this PR and raise a new one with a cleaner commit tree